PaddleCV/object_detection/INSTALL.md

```diff
@@ -0,0 +1,102 @@
+# Installing PaddleDetection
```

Comment:
I think instead of littering project root with `.md` files, we should add a `docs` folder

Reply:
Done